docs/src/userguide.md

```diff
@@ -28,22 +28,36 @@ To construct a sparse approximation to the exact posterior, we first need to sel
 M = 15  # The number of inducing points
 z = x[1:M]
 ```
-The inducing inputs `z` imply some latent function values `u = f(z)`, sometimes called pseudo-points. The stochastic variational Gaussian process (SVGP) approximation is defined by a variational distribution `q(u)` over the pseudo-points. In the case of GP regression, the optimal form for `q(u)` is a multivariate Gaussian, which is the only form of `q` currently supported by this package.
+The inducing inputs `z` imply some latent function values `u = f(z)`, sometimes called pseudo-points. The `SparseVariationalApproximation` specifies a distribution `q(u)` over the pseudo-points. In the case of GP regression, the optimal form for `q(u)` is a multivariate Gaussian, which is the only form of `q` currently supported by this package.
 ```julia
 using Distributions, LinearAlgebra
 q = MvNormal(zeros(length(z)), I)
 ```
 Finally, we pass our `q` along with the inputs `f(z)` to obtain an approximate posterior GP:
 ```julia
 fz = f(z, 1e-6)  # 'observe' the process at z with some jitter for numerical stability 
-approx = SVGP(fz, q)  # Instantiate everything needed for the svgp approximation
+approx = SparseVariationalApproximation(fz, q)  # Instantiate everything needed for the approximation
```

Comment:
Hm, I guess this was missed in #70 because it's not actually a doctest... is that something that could be (reasonably) changed ?

Reply:
I don't know whether it's possible to add doctests to the docs directly (I think they have to be done in the docstring), but I might be wrong. There's definitely a case to be made that this entire doc should be a Literate script that we run every time we build the docs though.

Reply:
I've opened an issue about this as I'd rather do it as a follow up.

https://github.com/JuliaGaussianProcesses/ApproximateGPs.jl/issues/72